pkg/apiserver/admission/chain_builder.go

```diff
@@ -117,7 +117,7 @@ var (
 	// combinedAdmissionControlPlugins gives the in-order default admission chain for all resources resources.
 	// When possible, this list is used.  The set of openshift+kube chains must exactly match this set.  In addition,
 	// the order specified in the openshift and kube chains must match the order here.
-	CombinedAdmissionControlPlugins = []string{
+	combinedAdmissionControlPlugins = []string{
```

Comment:
This used by anything other than tests?

Reply:
Nope, but the tests are verifying order and origin plugins. I initially thought about removing it entirely but figured that due to tests I'll just make it private. It ensure that openshfit has appropriate list of plugins enabled. I could move it to tests explicitly, though.